nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
```

Comment:
- [ ] Good job adding the error message to Nameable class. I think it will be better if you expand this error message to be specific to the class so that users can understand fully where the error is coming from 👍🏽 

you can consider :

```
 def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```